scripts/dev.ts

```diff
@@ -135,10 +176,12 @@ class DevRunner extends events.EventEmitter {
   async run() {
     process.env.NODE_ENV = 'development';
     try {
+      await this.startRendererProcess();
+
       await buildUtils.wait(
```

Comment:
If there's only one thing, there's no point to use `buildUtils.wait` is there?

Reply:
Fixed